web/styles/widgets.css

```diff
@@ -209,7 +209,7 @@ button {
         padding: 4px;
         padding-left: 14px;
         padding-right: 14px;
-        transition: all 0.2s ease;
+        transition: all 0.1s ease;
```

Comment:
Let's keep the old transition value but move it to `:hover, :focus and :active` selectors below. That should make independent of the theme switch transition.

Reply:
Hi @amanagr,

I've moved the transition duration to the :hover, :focus, and :active selectors for the .add-task and .poll-option buttons to ensure it is independent of the theme switch transition. Please review the updated changes.